cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,7 @@
+describe('Open the website', () => {
+  it('passes', () => {
+    cy.visit('http://work.co/')
+
+    cy.contains('We design and ship digital products that transform companies.')
```

Comment:
`cy.contains` only finds the element containing that text - [doc](https://docs.cypress.io/api/commands/contains).
For assertion we need to use should - [doc](https://docs.cypress.io/api/commands/should#Syntax)

Reply:
@IsabelaRosolen  still requires this change

Reply:
Thank you, @stefanworkco. Just added the changes.